package.json

```diff
@@ -22,12 +22,16 @@
     "test": "yarn lint && yarn format && git diff --quiet"
   },
   "peerDependencies": {
-    "@babel/eslint-parser": "^7.15.0",
-    "eslint": "^8.1.0",
+    "@babel/core": "7.16.0",
+    "@babel/eslint-parser": "7.16.3",
+    "@babel/plugin-transform-react-jsx-development": "7.16.0",
+    "eslint": "8.3.0",
```

Comment:
Peers should be hat-range dependencies left at the earliest supported version. For eslint-config that should be:
- @babel/eslint-parser: ^7.15.0
- eslint: ^8.1.0

Reply:
Fixed in https://github.com/zendeskgarden/eslint-config/pull/181/commits/e568f39d835a95156a53749cd594478d244e1388